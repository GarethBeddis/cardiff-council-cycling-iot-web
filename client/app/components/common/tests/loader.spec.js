import React from 'react'
import { shallow } from 'enzyme'

import Loader from '../loader'

describe('Loader Component', () => {
  // General Tests
  it('Renders without crashing', () => {
    shallow(<Loader />)
  })
  // Snapshot Tests
  describe('Snapshots', () => {
    it('Loader matches Snapshot', () => {
      let wrapper = shallow(<Loader />)
      expect(wrapper).toMatchSnapshot()
    })
  })
})
