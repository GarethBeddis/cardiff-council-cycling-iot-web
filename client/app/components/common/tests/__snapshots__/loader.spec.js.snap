// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader Component Snapshots Loader matches Snapshot 1`] = `
<div
  className="loader-container"
>
  <div
    className="loader"
  >
    <svg
      className="circle"
      viewBox="0 0 50 50"
    >
      <circle
        className="path"
        cx={25}
        cy={25}
        fill="none"
        r={20}
        strokeMiterlimit={10}
        strokeWidth={3}
      />
    </svg>
  </div>
</div>
`;
